src/views/Home/components/RoadmapSection/useRoadmapSection.tsx

```diff
@@ -21,12 +21,24 @@ const useRoadmapSection = (roadmapsData: Roadmap[]) => {
     }
   };
 
+  const coordinatorsRef = useRef<HTMLDivElement[]>([]);
+
+  useLayoutEffect(() => {
```

Comment:
For this case, do you see any issue with using `useEffect` instead? ` useLayoutEffect` tends to affect the performance and block the browser to paint the UI, so if we can avoid it, would be the best

Reply:
Changed to useEffect :)